client/src/App.js

```diff
@@ -1,17 +1,22 @@
 import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route } from "react-router-dom";
-
 import { theme } from "./themes/theme";
 import LandingPage from "./pages/Landing";
+import TempLanding from "./pages/TempLanding";
+import Signup from "./pages/Dialogs/Signup";
+import Signin from "./pages/Dialogs/Login";
 
 import "./App.css";
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route path="/" component={LandingPage} />
+        {/* <Route path="/" component={Landing} /> */}
+        {/* <Route path="/home" component={TempLanding} /> */}
```

Comment:
Don't leave commented out code - you can always delete and use git to see what you had in the past